src/shared/utilities/vsCodeUtils.ts

```diff
@@ -40,3 +41,34 @@ export function folderIconPath(): vscode.ThemeIcon | { light: vscode.Uri; dark:
         return vscode.ThemeIcon.Folder
     }
 }
+
+/**
+ * Executes the close all editors command and waits for all visible editors to disappear
+ */
+export async function closeAllEditors() {
+    await vscode.commands.executeCommand('workbench.action.closeAllEditors')
+
+    // The output channel counts as an editor, but you can't really close that...
+    const noVisibleEditor: boolean | undefined = await waitUntil(
+        async () => {
+            const visibleEditors = vscode.window.visibleTextEditors.filter(
+                editor => !editor.document.fileName.includes('extension-output')
+            )
+
+            return visibleEditors.length === 0
+        },
+        {
+            timeout: 2500,
```

Comment:
Can you add a comment about these arbitrary numbers (are they arbitrary?)

Reply:
Yes I can (they are arbitrary, unfortunately). Trying to figure out a cleaner way to do it.

Reply:
Arbitrary is fine, just add a comment that it is